assets/js/base/hooks/customer/use-customer-data.js

```diff
@@ -19,73 +20,94 @@ export const useCustomerData = () => {
 	const { updateCustomerData } = useDispatch( storeKey );
 	const { addErrorNotice, removeNotice } = useStoreNotices();
 	const {
-		billingAddress: initialBillingData,
-		shippingAddress: initialShippingAddress,
+		billingAddress: cartBillingData,
```

Comment:
I'm getting a typedef warning here for property `billingAddress` not existing for type `StoreCart`.

Reply:
I think you need to refresh your IDE; it's definitely defined.

Reply:
ahh okay, maybe it was due to me switching branches or something then.